manifests/z/ZelloInc/Zello/16.1.0.0/ZelloInc.Zello.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ZelloInc.Zello
+PackageVersion: 16.1.0.0

```

Comment:
Not sure whether `16.1.0.0` is the best `PackageVersion` if it's writing `2.6.0.0` to registry. It would probably be best to replace the [existing](https://github.com/microsoft/winget-pkgs/tree/master/manifests/z/ZelloInc/Zello/2.6.0.0) `2.6.0.0` version.